cupy/padding/pad.py

```diff
@@ -2,15 +2,38 @@
 
 
 def pad(array, pad_width, mode=None, **kwargs):
-    if not isinstance(pad_width, int):
+    if mode != 'constant':
         raise NotImplementedError
 
-    if mode != 'constant':
+    if kwargs.keys() == []:
+        fill_value = 0
+    elif kwargs.keys() == ['constant_values']:
+        if not isinstance(kwargs['constant_values'], int):
+            raise NotImplementedError
+        fill_value = kwargs['constant_values']
+    else:
         raise NotImplementedError
 
-    shape = tuple(s + pad_width * 2 for s in array.shape)
-    # TODO(unno): Fix fill value
-    ret = cupy.full(shape, 0, dtype=array.dtype)
-    index = tuple(slice(pad_width, pad_width + s) for s in array.shape)
-    ret[index] = array
+    if isinstance(pad_width, int):
+        shape = tuple(s + pad_width * 2 for s in array.shape)
+        ret = cupy.full(shape, fill_value, dtype=array.dtype)
+        index = tuple(slice(pad_width, pad_width + s) for s in array.shape)
```

Comment:
note: `tuple(f(x) for x in xs)` is slower than `tuple([f(x) for x in xs])`. List comprehension is faster.